.github/workflows/static_checks.yml

```diff
@@ -58,3 +58,45 @@ jobs:
       run: |
         echo "$GITCOOKIE_SH" | bash
         bin/fmt
+  shellcheck:
+    name: shellcheck
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: shellcheck
+      # TODO: fix our shell scripts and re-enable checks
+      run: |
+        bin/shellcheck bin/* --exclude="\
```

Comment:
Without doing a bunch of research, I'm not sure how to approach or understand this list of exclusions.  I assume that some of these are excluded because we would like to fix them, but don't want to do it right now.  Whereas perhaps others are excluded because we don't agree with the rule.  

I'm not sure how, but it would be great if this list of exclusions was more understandable somehow.  If a contributor wanted to help burn this down, it's now clear where to start.

Reply:
Would it make sense to exclude files instead of excluding rules?  That would ensure that all new scripts are complaint with all rules and give us perhaps an easier "TODO list" of existing files to fix up.

Reply:
This list of exclusions is exactly what's necessary to make `shellcheck` pass, I didn't evaluate any of the rules based on whether we _should_ use them.

I like the idea of excluding files instead, let me take a look at that.